src/test/java/com/ait/lienzo/client/widget/LienzoHandlerManagerTest.java

```diff
@@ -156,36 +156,26 @@ public void testMouseDownHandler_RightButton() {
         final boolean isLeft = false;
         final boolean isMiddle = false;
         final boolean isRight = true;
-        final MouseDownEvent event = mock(MouseDownEvent.class);
-        doReturn(NativeEvent.BUTTON_RIGHT).when(event).getNativeButton();
-
-        verify(lienzoPanel).addMouseDownHandler(mouseDownHandler.capture());
-
-        mouseDownHandler.getValue().onMouseDown(event);
-
-        verify(dragMouseControl).allowDrag(isLeft, isMiddle, isRight);
+        verifyMouseDownHandlerButton(isLeft, isMiddle, isRight);
     }
 
     @Test
     public void testMouseDownHandler_LeftButton() {
         final boolean isLeft = true;
         final boolean isMiddle = false;
         final boolean isRight = false;
-        final MouseDownEvent event = mock(MouseDownEvent.class);
-        doReturn(NativeEvent.BUTTON_LEFT).when(event).getNativeButton();
-
-        verify(lienzoPanel).addMouseDownHandler(mouseDownHandler.capture());
-
-        mouseDownHandler.getValue().onMouseDown(event);
-
-        verify(dragMouseControl).allowDrag(isLeft, isMiddle, isRight);
+        verifyMouseDownHandlerButton(isLeft, isMiddle, isRight);
     }
 
     @Test
     public void testMouseDownHandler_MiddleButton() {
         final boolean isLeft = false;
         final boolean isMiddle = true;
         final boolean isRight = false;
+        verifyMouseDownHandlerButton(isLeft, isMiddle, isRight);
+    }
+
+    private void verifyMouseDownHandlerButton(final boolean isLeft, final boolean isMiddle, final boolean isRight) {
```

Comment:
This refactor is only partially complete.

You need to also pass a parameter to setup ```doReturn(NativeEvent.XXX).when(event).getNativeButton()```